### ISSUE TITLE
Support NewRecord expressions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SwensenSoftware/unquote#110]
```
Support Patterns.NewRecord expressions. This should be pretty straight forward.
```

Original issue reported on code.google.com by `stephen....@gmail.com` on 23 Mar 2015 at 6:38

**Comment by thinkbeforecoding** [comment]
This is addressed by #150 👍

**Comment by stephen-swensen** [comment]
Complete and merged into fs46 branch / PR
